src/main/java/frc/team2412/robot/commands/intake/IntakeCommand.java

```diff
@@ -0,0 +1,32 @@
+package frc.team2412.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.team2412.robot.subsystem.IndexSubsystem;
+import frc.team2412.robot.subsystem.IntakeSubsystem;
+
+public class IntakeCommand extends CommandBase {
+
+    private IntakeSubsystem intakeSubsystem;
+    private IndexSubsystem indexSubsystem;
+
+    public IntakeCommand(IntakeSubsystem intakeSubsystem, IndexSubsystem indexSubsystem) {
+        this.intakeSubsystem = intakeSubsystem;
+        this.indexSubsystem = indexSubsystem;
+        addRequirements(intakeSubsystem);
```

Comment:
I assume we aren't requiring the index subsystem because we're only reading from it?

Reply:
Yes